packages/astro/src/assets/vite-plugin-assets.ts

```diff
@@ -42,7 +42,7 @@ export default function assets({
 				extendManualChunks(outputOptions, {
 					after(id) {
 						if (id.includes('astro/dist/assets/services/')) {
-							return `astro-assets-services`;
+							return `astro/assets-service`;
```

Comment:
is this for some convention we have, that everything astro core is in an `astro/` folder?

Reply:
Not an established convention, but yes, I was thinking it would be nice to organize different parts of the Astro server code in a single `astro/` directory.